test/TensorFlowNET.UnitTest/PythonTest.cs

```diff
@@ -196,17 +245,25 @@ public T evaluate<T>(Tensor tensor)
             //    return self._eval_helper(tensors)
             //  else:
             {
-                var sess = tf.Session();
+                var sess = tf.get_default_session();
```

Comment:
@Wanglongzhi2001 The same functions exist in another assembly. 
Should we generalize them in one common assembly in another PR?

Reply:
I think so, if you would like to.